internal/pkg/cli/deploy/deploy.go

```diff
@@ -1211,6 +1210,8 @@ func validateAppVersionForAlias(appName string, appVersionGetter versionGetter)
 
 func (d *backendSvcDeployer) validateALBRuntime() error {
 	switch {
+	case d.backendMft.RoutingRule.IsEmpty():
+		return nil
```

Comment:
What do you think of:
```
if d.backendMft.RoutingRule.IsEmpty() {
   return nil
}
switch {
}
```
leaving `switch` to handle enumerating the 4 possible cases for alias x imported cert  only.